specs/zipWith.test.ts

```diff
@@ -1,27 +1,33 @@
-import { describe, it } from './_describe.ts'
-import { zipWith, add, multiply, _ } from '../mod.ts'
-import { eq } from './utils/utils.ts'
+import { describe, it } from "./_describe.ts";
+import { zipWith, add, multiply, _ } from "../mod.ts";
+import { eq } from "./utils/utils.ts";
+import { filter } from "../filter.ts";
 
-describe('zipWith', () => {
+describe("zipWith", () => {
```

Comment:
we are using single quotes every where, please run prettier before push 
`prettier --write .`